File: PHP/StringToCamelCase/MySolution.php
<?php
function toCamelCase($str){
  $arrayString = explode("_", str_replace("-", "_", $str));
  $counterSolution = "";
  foreach ($arrayString as $word) {
    $counterSolution .= $counterSolution != "" ? ucfirst($word) : $word; 
  }
  return $counterSolution;
}

// Original kata: https://www.codewars.com/kata/517abf86da9663f1d2000003
// My solution: https://www.codewars.com/kata/reviews/5af529b442d45e8ca300134e/groups/67863f77c90009c787b80e76

File: CS/IfElse#7/MySolution.cs
public class Kata
{
  public static int SaleHotDogs(int n)
  {
    switch (n)
    {
        case < 5:
            return n * 100;
        case < 10:
            return n * 95;
        case >= 10:
            return n * 90;
    }
  }
}

// Original kata: https://www.codewars.com/kata/57202aefe8d6c514300001fd
// My solution: https://www.codewars.com/kata/reviews/59af4560ff87fd540e0004b3/groups/67f8b37abc0e087def189be6
